mvrx-rxjava2/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -146,6 +147,18 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
      */
     protected fun subscribe(subscriber: (S) -> Unit): Disposable = onEachInternal(null, action = { subscriber(it) }).toDisposable()
 
+    /**
+     * Subscribe to state when this LifecycleOwner is started.
+     */
+    @RestrictTo(RestrictTo.Scope.LIBRARY)
```

Comment:
i'm a bit confused, did you mean to restrict this to the library? I thought you wanted to expose it for you?

Reply:
@elihart I left it with the same visibility as it is in 1.0. As mentioned, for some reason, Android Studio never showed us the warning for this.

Reply:
@gpeal if I remember correctly, lint started showing this warning after upgrading to agp 4.0